bin/server/cmd/root.go

```diff
@@ -98,7 +99,14 @@ var (
 		pgURL string
 		// disableMetric is the flag to disable the metric collector.
 		disableMetric bool
+
+		// Cloud backup configs
+		backupRegion     string
+		backupBucket     string
+		backupCredential string
```

Comment:
Should backupStorageBackend be part of this?

Reply:
It's not directly passed in as a flag, but parsed from backupBucket (default to s3 for now).

Reply:
This looks strange in this case. Set this flag as part of getBaseProfile in this case.

Reply:
Done.